pkg/common/utils/utils.go

```diff
@@ -180,6 +180,9 @@ func PodUnderCondition(pod *v1.Pod, condition *v1.PodCondition) bool {
 func GetNamespaceGuaranteedFromAnnotation(namespaceObj *v1.Namespace) *si.Resource {
 	// retrieve guaranteed resource info from annotations
 	namespaceGuaranteed := GetNameSpaceAnnotationValue(namespaceObj, constants.NamespaceGuaranteed)
```

Comment:
I believe this is more Go-like:
```
if guaranteed := GetNameSpaceAnnotationValue(namespaceObj, constants.NamespaceGuaranteed); guaranteed != "" {
  ...
}
return nil
```

No need to name it "namespaceGuaranteed", it's pretty obvious from the context. Prefer shorter variables if possible.

Reply:
Thanks for reviewing my code and the helpful recommendation.  I'll make that change to implement your feedback.